initializers/ah-swagger-plugin.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const { Initializer, api } = require('actionhero')
+
+module.exports = class SwaggerPlugin extends Initializer {
+  constructor () {
+    super()
+    this.name = 'swaggerJson'
```

Comment:
maybe rename this to just `swagger`?  The initializer doesn't really have an opinion on the format of the output